.eslintrc

```diff
@@ -1,51 +1,17 @@
 {
   "parserOptions": {
-    "ecmaVersion": 2017
+    "ecmaVersion": 2015
```

Comment:
Why have we gone backwards here?

Reply:
This is a node 6 project ('cos it's used by the CLI), so it has to be valid es2015. May as well catch it at linting time, instead of later!

...for another 15 days, and then we'll be changing this not to support node 6, and changing it back to 2017, I guess.